web/api/web_api_v1.c

```diff
@@ -934,10 +941,285 @@ inline void host_labels2json(RRDHOST *host, BUFFER *wb, size_t indentation) {
+    if (host->system_info->container_os_id)
+        JSON_ADD_STRING("container_os_id", host->system_info->container_os_id, j)
+    if (host->system_info->container_os_id_like)
+        JSON_ADD_STRING("container_os_id_like", host->system_info->container_os_id_like, j)
+    if (host->system_info->container_os_version)
+        JSON_ADD_STRING("container_os_version", host->system_info->container_os_version, j)
+    if (host->system_info->container_os_version_id)
+        JSON_ADD_STRING("container_os_version_id", host->system_info->container_os_version_id, j)
+    if (host->system_info->container_os_detection)
+        JSON_ADD_STRING("container_os_detection", host->system_info->container_os_detection, j)
+    if (host->system_info->is_k8s_node)
+        JSON_ADD_STRING("is_k8s_node", host->system_info->is_k8s_node, j)
+
+    JSON_ADD_STRING("kernel_name", (host->system_info->kernel_name) ? host->system_info->kernel_name : "", j)
+    JSON_ADD_STRING("kernel_version", (host->system_info->kernel_version) ? host->system_info->kernel_version : "", j)
+    JSON_ADD_STRING("architecture", (host->system_info->architecture) ? host->system_info->architecture : "", j)
+    JSON_ADD_STRING("virtualization", (host->system_info->virtualization) ? host->system_info->virtualization : "", j)
+    JSON_ADD_STRING("virt_detection", (host->system_info->virt_detection) ? host->system_info->virt_detection : "", j)
+    JSON_ADD_STRING("container", (host->system_info->container) ? host->system_info->container : "", j)
+    JSON_ADD_STRING("container_detection", (host->system_info->container_detection) ? host->system_info->container_detection : "", j)
+
+    json_object_object_add(j, "host_labels", host_labels_json(host));
+
+    json_object_object_add(j, "collectors", chartcollectors_json(host));
+
+#ifdef DISABLE_CLOUD
+    JSON_ADD_BOOL("cloud-enabled", 0, j)
+#else
+    JSON_ADD_BOOL("cloud-enabled", appconfig_get_boolean(&cloud_config, CONFIG_SECTION_GLOBAL, "enabled", 1), j)
+#endif
+
+#ifdef ENABLE_ACLK
+    JSON_ADD_BOOL("cloud-available", 1, j)
+    JSON_ADD_BOOL("aclk-ng-available", 1, j)
+#ifdef ENABLE_NEW_CLOUD_PROTOCOL
+    JSON_ADD_BOOL("aclk-ng-new-cloud-protocol", 1, j)
+#else
+    JSON_ADD_BOOL("aclk-ng-new-cloud-protocol", 0, j)
+#endif
+    JSON_ADD_BOOL("aclk-legacy-available", 0, j)
+    JSON_ADD_STRING("aclk-implementation", "Next Generation", j)
+#else /* !ENABLE_ACLK */
+    JSON_ADD_BOOL("cloud-available", 0, j)
+#endif /* ENABLE_ALCK */
+
+    char *agent_id = is_agent_claimed();
+    JSON_ADD_BOOL("agent-claimed", (agent_id != NULL), j)
+    freez(agent_id);
+
+#ifdef ENABLE_ACLK
+    if (aclk_connected) {
+        JSON_ADD_BOOL("aclk-available", 1, j)
+#ifdef ENABLE_NEW_CLOUD_PROTOCOL
+        JSON_ADD_STRING("aclk-available-protocol", aclk_use_new_cloud_arch ? "New" : "Legacy", j);
+#else
+        JSON_ADD_STRING("aclk-available-protocol", "Legacy", j);
+#endif
+    } else
+#endif
+        JSON_ADD_BOOL("aclk-available", 0, j)
```

Comment:
Please, fix tabulation here.

Reply:
i consider it to be correct due to else on the line 1164? do you prefer it one tab left?

Reply:
I am sorry, you can ignore this, please, only centralize the other defines.